react/__tests__/MiniCart.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { render } from 'test-utils'
+import { fireEvent } from 'react-testing-library'
+
+import MiniCart from './../index'
+
+describe('<MiniCart /> component', () => {
+  const renderComponent = customProps => {
```

Comment:
**Suggestion**: I prefer tests that are repetitive and easy to change than using functions that "make tests" or keep them DRY.

So in this case I would use the render function directly. It's also more explicit what you are trying to test.

```js
render(<MiniCart type="popup" hideContent={false} />)
```